### ISSUE TITLE
Receiving 404 from sendmail.php

### ISSUE THREAD

**Issue by dustinsummers** [themefisher/timer-hugo#48]
Hello,

I have setup the Contact Form according to your example, by adding in my formspree to the `contactFormAction` inside of the `config.toml`.  However, when I attempt to use the form, I get a 404 error.

Are you all able to assist with this?

See the error below for details:

![Screen Shot 2020-09-25 at 6 55 05 AM](https://user-images.githubusercontent.com/25121655/94259291-3c978c80-fefc-11ea-81ba-283b3ad14e8c.png)

Thanks!

**Comment by divinerites** [comment]
The most probable is that you're hosting your site where there is no PHP server/page sendmail.php.

**Comment by dustinsummers** [comment]
Ah, that sounds like it could be the case.  I'm hosting via Firebase.  Any suggestions on how to work around that?

**Comment by developer-evan** [comment]
hey @dustinsummers 

Thanks for your concern. This contact form works with https://formspree.io 
please download the latest code and follow this [doc](https://docs.gethugothemes.com/guide/#contact-and-subscription-form)

**Comment by dustinsummers** [comment]
Hey @developer-evan and @divinerites, I was able to fix my issue!

I know @divinerites edited his original comment, but he was right.  Since I'm hosting through Firebase Hosting, it had no way to run the `sendmail.php` code.  

Instead, I needed to add in firebase cloud functions to my site as well, and copy over your contact-form code into there.  

So it's working now.  Thanks again!  Donation coming your way shortly :) 

- Dustin
